content/en/blog/_posts/2021-04-15-local-storage-features-go-beta.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: blog
+title: 'Local Storage: Storage Capacity Tracking, Distributed Provisioning and Generic Ephemeral Volumes hit Beta
+date: 2021-04-15
+slug: 2021-04-15-local-storage-features-go-beta
+---
+
+ **Authors:** Patrick Ohly (Intel)
+
+Several features are getting promoted to beta in Kubernetes 1.21 and
```

Comment:
So "generic ephemeral volumes" and "storage capacity tracking" are clearly getting promoted now and can be named here.

I'm less sure how to describe "distributed provisioning". It was a new feature in external-provisioner v2.1.0. I suppose because it wasn't declared as "alpha" at that time, it immediately became a stable feature? We don't really have a stability tracking process per feature in the sidecars.

Reply:
I've rewritten the introduction.